### ISSUE TITLE
Update visibility simulation-related tutorials

### ISSUE THREAD

**Issue by r-pascua** [HERA-Team/hera_sim#236]
Some of the tutorials have broken with the recent update that removed the option to use pixel beams in `vis_cpu`, so we should fix them.

**Comment by steven-murray** [comment]
Really? I thought I updated everything.

**Comment by r-pascua** [comment]
Yep. See e.g. cell 9 in the `visibility_simulator` tutorial. It actually looks like none of the tutorial notebooks were updated in #233. I'd suggest that this is a good time to revisit #84, but I don't really have the time to be working on that...

**Comment by hughbg** [comment]
I've updated the notebooks to run with the current code (updated-notebooks branch). There are a few notes about this:

- I got them running but that doesn't mean that they still mean the same thing. Sometimes I had to change parameters, change code, so the meaning may have also changed. In particular, the RFI in the hera_sim_tour is going to be wrong.
- I compared all the plots between old/new and none of them have changed radically, except for the RFI. However, they are slightly different, which could be due to code changes, not using pixel beam anymore, randomness, or ....
- There are more warnings than there used to be.
- "Works for me". I can run them all but others should check.

I could make a pull request but the RFI in hera_sim_tour needs more work.

**Comment by steven-murray** [comment]
@hughbg please make a PR (even if it's just a Draft). That way we can continue discussion there.
